jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -553,6 +553,7 @@ def _initialize_model_component(self, model_comp, comp_label, poly_order=None):
             # equivs for spectral density and flux<>sb
             pixar_sr = masked_spectrum.meta.get('_pixel_scale_factor', 1.0)
             equivs = all_flux_unit_conversion_equivs(pixar_sr, init_x)
+            equivs += u.spectral_density(init_x)
```

Comment:
should this be in `all_flux_unit_conversion_equivs` (or at least as an additional option)?

Reply:
Yes, it should already be handled by `all_flux_unit_conversion_equivs`, I was debugging an issue and threw it in there, just retested and it is able to be removed with no consequence.